src/containers/Notifications.js

```diff
@@ -32,8 +32,13 @@ class Notifications extends Component {
 	handleSubmit(event) {
 		event.preventDefault();
 
-		axios
-			.put(`${config.API_BASE_URL}/users/me/`)
+		axios({
+            method: 'put',
```

Comment:
should i use tabs instead of spaces here? it looks different on my editor when in github...

Reply:
I use tabs personally

Reply:
Is there a generic way to set headers?
https://github.com/axios/axios/issues/209
https://stackoverflow.com/questions/43051291/attach-authorization-header-for-all-axios-requests

Reply:
I could work on this in a separate PR.